e2e/client/specs/internal_destinations_spec.ts

```diff
@@ -155,28 +155,26 @@ describe('internal destinations & generic-page-list', () => {
         expect(el(['gform-output--name'], null, items.get(1)).getText()).toBe('charlie');
     });
 
-    it('can display and remove active filters', () => {
-        var items = els(['list-page--items', 'internal-destinations-item']);
-
-        browser.wait(ECE.hasElementCount(items, 3));
+    fit('can display and remove active filters', () => {
```

Comment:
Was this the test that was failing? This is the one that's quite good compared to other protractor tests. Did you find the root cause of why was it failing?

Reply:
This was the test yes. The root cause I suspect is in the autocomplete component re-rendering multiple times, so after you get it with a selector in the test the reference can't be used after that same component re-renders. This is why we were getting a stale element exception